src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts

```diff
@@ -12,34 +12,46 @@ import {
 import { DeviceRegistryEntry } from "../../../../../../data/device_registry";
 import { haStyle } from "../../../../../../resources/styles";
 import { HomeAssistant } from "../../../../../../types";
-import { OZWDevice, fetchOZWNodeStatus } from "../../../../../../data/ozw";
+import {
+  OZWDevice,
+  fetchOZWNodeStatus,
+  getIdentifiersFromDevice,
+  OZWNodeIdentifiers,
+} from "../../../../../../data/ozw";
+import { showOZWRefreshNodeDialog } from "../../../../integrations/integration-panels/ozw/show-dialog-ozw-refresh-node";
 
 @customElement("ha-device-info-ozw")
 export class HaDeviceInfoOzw extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
   @property() public device!: DeviceRegistryEntry;
 
+  @property()
+  private node_id = 0;
+
+  @property()
+  private ozw_instance = 1;
+
   @internalProperty() private _ozwDevice?: OZWDevice;
 
   protected updated(changedProperties: PropertyValues) {
     if (changedProperties.has("device")) {
-      this._fetchNodeDetails(this.device);
+      const identifiers: OZWNodeIdentifiers | false = getIdentifiersFromDevice(
+        this.device
+      );
+      if (!identifiers) return;
```

Comment:
Please always use curly braces for `if` blocks
```suggestion
      if (!identifiers) {
        return;
      }
```